### ISSUE TITLE
Small changes to view service page

### ISSUE THREAD

**Issue by iRomain** [PleaseDotCom/frontend#271]
Example: https://please-master.now.sh/#/services/5bb0e21f93e31066109c16ae

remove the whole "instructions" section (the word "instructions", as well as the content below)
make subtitle in bold
working hours is not showing the right times
make the location (eg. George Town, KY) a link to the search (/results?address=London,%20UK&latitude=51.5073509&longitude=-0.12775829999998223)

**Comment by iRomain** [comment]
@jaydp17 Is the API sending the lat/long to be used for the link?

**Comment by jaydp17** [comment]
@iRomain yes, it's there in the API response on path `location.geo.coordinates`
